code.swift

```diff
@@ -54,25 +54,24 @@ class PeriodicTableViewController: UIViewController {
     }
     
     // This next method was added by a new member of the team. Identify the 10 formatting, naming, and logic issues below:
-    @IBAction func chooseRandomElement(_ sender: Any){
-        let ranElem = elementsContoller.elements.randomElement()
+    @IBAction func chooseRandomElement(_ sender: Any) {
+        let ranElem = elementsContoller.elements
         
-        var chosenElementView:ElementView!
+        var chosenElementView: ElementView
         
-        for e in elementViews {
-            if (e.element == ranElem) {
-                chosenElementView = e
+        for elementView in elementViews  {
```

Comment:
Why the extra space all of a sudden? You are supposed to remove formatting issues, not add new ones 😛